packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -174,14 +172,36 @@ export function buildRootObject(powers, _params, testJigSetter = undefined) {
       zcfSeatToZCFSeatAdmin.init(zcfSeat, zcfSeatAdmin);
       zcfSeatToSeatHandle.init(zcfSeat, seatHandle);
 
+      return {
+        updater,
+        zoeSeatAdminPromiseKit,
+        seatHandle,
+        zcfSeat,
+      };
+    }
+
+    const makeEmptySeatKit = (exit = { onDemand: null }) => {
+      const proposal = cleanProposal(getAmountMath, harden({ exit }));
+      const {
+        updater,
+        zoeSeatAdminPromiseKit,
+        seatHandle,
+        zcfSeat,
+      } = initializeEmptySeatData(proposal);
+
       const exitObj = makeExitObj(
-        seatData.proposal,
+        proposal.exit,
         zoeSeatAdminPromiseKit.promise,
-        zcfSeatAdmin,
+        zcfSeatToZCFSeatAdmin.get(zcfSeat),
       );
 
+      const userSeatPromiseKit = makePromiseKit();
+      // Don't trigger Node.js's UnhandledPromiseRejectionWarning.
+      // This does not suppress any error messages.
+      userSeatPromiseKit.promise.catch(_ => {});
+
       E(zoeInstanceAdmin)
-        .makeNoEscrowSeat(initialAllocation, proposal, exitObj, seatHandle)
+        .makeNoEscrowSeat({}, proposal, exitObj, seatHandle)
```

Comment:
Note that an empty object `{}` will be marshalled as a presence.  It won't have any ownProperties on the other side, but this will be surprising if that presence is exposed to user code.